src/main/java/seedu/address/logic/parser/ExportCommandParser.java

```diff
@@ -16,7 +18,7 @@
     public ExportCommand parse(String args) throws ParseException {
         String fileName = args.trim();
 
-        if (!fileName.endsWith(".json") || fileName.equals(".json")) {
+        if (!fileName.endsWith(".json") || fileName.equals(".json") || !isValidJsonFileName(fileName)) {
```

Comment:
Not a bug. Having && is easier to follow than ||. Perhaps we can use De Morgans to convert || to &&

Reply:
I think this one is still quite understandable